docs/circuits.ipynb

```diff
@@ -579,6 +579,137 @@
     "Especially useful is dropping the last moment (which are often just measurements): `circuit[:-1]`, or reversing a circuit: `circuit[::-1]`.\n"
```

Comment:
<p>Do you need to treat a parameterized circuit differently if it has a circuit operation in it? If the frozen subcircuit inside a circuit operation has symbolic parameters in it, does the CircuitOperation itself keep track of that? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5399/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
I didn't go into parameterization here because it's a deeper topic that doesn't appear on this (relatively high-level) doc, but the answer is that `CircuitOperation` knows what parameters are used inside it for the purposes of e.g. `cirq.is_parameterized`, and has arguments to re-parameterize its contained circuit (either to remap parameter names or assign values to them).

Reply:
<p>Makes sense to me, thanks. </p>